src/game/Player.cpp View file @ d8eecdf
void Player::RemoveItem(uint8 bag, uint8 slot, bool update)
@@ -20903,7 +20921,7 @@ AreaLockStatus Player::GetAreaTriggerLockStatus(AreaTrigger const* at, uint32& m
      if (map && map->IsDungeon())
      {
          // cannot enter if the instance is full (player cap), GMs don't count
-         if (((DungeonMap*)map)->GetPlayersCountExceptGMs() >= ((DungeonMap*)map)->GetMaxPlayers())
+ 	      if (((DungeonMap*)map)->GetPlayersCountExceptGMs() >= ((DungeonMap*)map)->GetMaxPlayers() && ((DungeonMap*)map)->GetId() != GetMapId())
              return AREA_LOCKSTATUS_INSTANCE_IS_FULL;
  
          // In Combat check
